### ISSUE TITLE
JToolTip does not respect their location

### ISSUE THREAD

**Issue by juersimon** [JFormDesigner/FlatLaf#468]
As already noted in #188, the position of tooltips is not respected if they cover the component. The correction made only works if the tooltip was not created by the ToolTipManager. The test case included in #188 uses a somewhat unusual way of positioning the tooltip. More commonly, the position of the tooltip might be adjusted by overriding the getToolTipLocation method in JComponent. In this case the change has no effect.
I suggest to define another property in FlatClientProperties:
~~~java
String POPUP_DISABLE_FIX_LOCATION = "Popup.disableFixLocation";
~~~
And to check this in FlatPopupFactory:
~~~java
@Override
public Popup getPopup( Component owner, Component contents, int x, int y )
    throws IllegalArgumentException
{
    if (!isOptionEnabled(owner, contents, FlatClientProperties.POPUP_DISABLE_FIX_LOCATION, "Popup.disableFixLocation")) {
        Point pt = fixToolTipLocation( owner, contents, x, y );
...
~~~

I have adapted the test case accordingly.
[FlatLafToolTip.zip](https://github.com/JFormDesigner/FlatLaf/files/7896241/FlatLafToolTip.zip)

**Comment by DevCharly** [comment]
Thanks for pointing out this issue.

This is actually a bug and no additional option should be needed to fix a bug...

**Comment by DevCharly** [comment]
Should be fixed in latest `2.0.1-SNAPSHOT` https://github.com/JFormDesigner/FlatLaf#snapshots

Please give it a try
